pxr/usd/plugin/usdAbc/CMakeLists.txt

```diff
@@ -205,6 +211,15 @@ pxr_register_test(testUsdAbcIndexedProperties
         USD_ABC_TESTSUFFIX=def
 )
 
+pxr_register_test(testUsdAbcIndexedGeomArb
+        PYTHON
+        COMMAND "${CMAKE_INSTALL_PREFIX}/tests/testUsdAbcIndexedGeomArb"
+        DIFF_COMPARE bad_primvars_namespace_before_fix.usda
```

Comment:
Shouldn't we be comparing to the "good" , after-fix baseline, here?  And I don't think we need the "bad" version included in the PR - I only mentioned it so that you could have an extra validation that your fix works by running the test against the *good* baseline, *without your fix*, to see that the test fails, a la test-driven development.

Thanks, @BSalem !

Reply:
to be honest, I didn't ever work with tests modules, so I'm not familiar with their dependencies and how it work... I blindly followed the example you shared.
I just pushed an updated one though.